### ISSUE TITLE
cli/nats.go deleted

### ISSUE THREAD

**Issue by siredmar** [immutability-io/nkey#1]
I didn't find the `cli/nats.go` file on the current master.

Was the deletion of the file `cli/nats.go` intentionally in the latest commit (https://github.com/immutability-io/nkey/commit/5a9444a166be135b5507da805397f1560ba04d83)

**Comment by dnlopes** [comment]
Same issue here. The README also points to that file, which is now gone.

Is this expected? @cypherhat

**Comment by siredmar** [comment]
@dnlopes in the meantime we developed our own vault plugin for this matter. 

It can manage operators, accounts users, operator signing keys and account signing keys.
Feel free to use this. But keep in mind, that this is still very early stage.
https://github.com/edgefarm/vault-plugin-secrets-nats
If you need kubernetes native resource, you can also use our crossplane provider for this plugin:
https://github.com/edgefarm/provider-natssecrets

**Comment by dnlopes** [comment]
> @dnlopes in the meantime we developed our own vault plugin for this matter.
> 
> It can manage operators, accounts users, operator signing keys and account signing keys. Feel free to use this. But keep in mind, that this is still very early stage. https://github.com/edgefarm/vault-plugin-secrets-nats I you need kubernetes native resource, you can also use our crossplane provider for this plugin: https://github.com/edgefarm/provider-natssecrets

Yes, I found yours as well and so far so good :) I'm build a POC with it and Vault for my use case; the goal is to integrate your plugin with the newly launched VSO operator for K8s, so my K8s apps can fetch JTW tokens dynamically to publish messages to NATS.

Thanks for the project! ❤️

**Comment by siredmar** [comment]
@dnlopes star us if you like! 😊
